### PR TITLE
Answers : M-F Finished | Sprint : WIP | MyComprehension : ~65%

### PR DESCRIPTION
**Pull request by austie702**

### CODE REVIEW

File: Exercises.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    * to hex: 
+      * Divide 68 by 16 until the quotient is 0:
+      * 68 / 16 = 4, remainder is 4
+      * 4 / 16 = 0, remainder is 4
+      * Read from bottom (MSB) to top (LSB) as 44 = 0x44
+
+  #### Architecture
+
+  * Explain how the CPU provides concurrency: 
+    * Concurrency is a form of computing which allows several operations to be execuded in parralel, or concurrently, instead of sequentially.
```
Comment (adamlower) [inline Exercises.md]: True, just make sure not to get this confused with parallel computing.

File: project/ls8/cpu.js [hunk 2]
```diff
@@ -35,14 +39,18 @@ class CPU {
 	/**
 	 * Sets up the branch table
 	 */
+    // !!! IMPLEMENT ME
 	setupBranchTable() {
 		let bt = {};
```
Comment (adamlower) [inline project/ls8/cpu.js]: Remember to keep line spacing and indentation consistent.

**Comment by adamlower** [comment]
Overall great job on this, the fact that you decided to scrap everything and start over to reinforce the things you learned really shows how well you are starting to grasp these concepts. I recommend committing your code more frequently with descriptive messages. Keep up the great work!

**Comment by austie702** [comment]
Committing more frequently is a great idea for myself And Lambda. I'll try to remember to do that. It certainly doesn't come naturally to me but I like that plan so I'll work on it. Thanks for the crit!

**Pull request closed without merging**